extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -1135,7 +1138,9 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
   };
 
   private generateRecipientId = (): string => {
-    return `recipient_${this.addedRecipients.length}`;
+    const recipientId = `recipient_${this.uniqueRecipientIndex}`;
+    this.uniqueRecipientIndex += 1;
+    return recipientId;
```

Comment:
It seems unnecessary to store separate variable for unique index.

What about replacing it with
```js
private generateRecipientId = (): string => {
     const uniqueId = Math.floor(Math.random() * Date.now());
     return `recipient_${uniqueId}`;
}
```

Reply:
Well, I think generating random id would be not good for testing.

Reply:
Yes, you're right, then let's leave it as it is 👍